apps/website/gatsby-ssr.ts

```diff
@@ -27,4 +27,11 @@ export const onRenderBody: GatsbySSR['onRenderBody'] = ({
       // We don't know the language.
     }
   }
+  setHeadComponents([
+    React.createElement('link', {
+      key: 'styles.css',
+      rel: 'stylesheet',
+      href: '/styles.css',
```

Comment:
I assume this will introduce an issue with browser cache.

Would it be possible to change it to something like `/styles.css?hash={hash of styles.css contents}`?

Reply:
Hm. Actually this is probably a great improvement to the Netlify deploy time. Now Gatsby won't need to change all html files on any css change. Right?

Reply:
> I assume this will introduce an issue with browser cache.
Damn, thats really good catch.

> Hm. Actually this is probably a great improvement to the Netlify deploy time. Now Gatsby won't need to change all html files on any css change. Right?

In theory, yes. But css changes based on classes used in components, and if components change, it re-renders. But maybe we have less to upload.

Reply:
We can't do the hash, because the file is not yet there at that point. But i added a timestamp-cachebuster.

Reply:
Just realized that this means a new stylesheet bust on every content change. Will think more about it 😬

Reply:
Ah, wait, if we add content hash to the css url, there is no difference at all 😅

Reply:
> Just realized that this means a new stylesheet bust on every content change

A change to css url would re-upload all files to Netlify 🙈

Reply:
> We can't do the hash, because the file is not yet there at that point.

It's in the node_modules, no?

Reply:
Or! Could it be that I am completely wrong and https://www.npmjs.com/package/serve-static solves all the browser cache issues with magical headers?

Reply:
`serve-static` is only there during `gatsby dev`. 
Just tried to create the hash, and Gatsby says "unsafe usage of `readFileSync`, i'm going to rebuild EVERYTHING EVERY TIME".
Also we have a lot of other assets (html files), that update correctly. I think netlify takes care of that.

Reply:
Let's try without hash 🤞 I will test in all major browsers.

Reply:
The hash is back baby!
Using Gatsby standard behavior now. See commit messages.

Reply:
If we have component- and therefore stylesheet-change, we have to re-render and re-upload everything anyway. But having non-inline stylesheets makes more sense with tailwind.